src/bindfs.c

```diff
@@ -101,6 +101,7 @@
 /* Apple Structs */
 #ifdef __APPLE__
 #include <sys/param.h>
+#include <sys/dirent.h>
```

Comment:
@mpartel I tried this approach myself earlier, but for some reason it does not end up defining `DT_LNK` in practice. That's why I went for `#define DT_LNK 10` but it's not ideal, I agree. (I don't have access to macOS outside of CI myself.)